packages/next/server/dev/next-dev-server.ts

```diff
@@ -700,10 +700,18 @@ export default class DevServer extends Server {
     })
   }
 
+  protected getPagesManifest(): undefined {
```

Comment:
is `getPagesManifest` still required to be defined since it derives  from the next-server?
same with `getMiddlewareManifest`

Reply:
Yes since I refactored the caller conditions in the base server a bit so they'are always called: https://github.com/vercel/next.js/pull/32179/files?diff=unified&w=0#diff-2b8e3503c1bebf856610a5551d9c5996435b99f45db389d4eeac1b671518875eL306-R310

Goal is to eventually get rid of ad-hoc dev server logic from the base server.